src/Application/Messenger/Transport/SalesforcePushTransport.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace MatchBot\Application\Messenger\Transport;
+
+use Symfony\Component\Messenger\Transport\TransportInterface;
+
+/**
+ * This needs to be distinct from the service we inject with `TransportInterface` generically
+ * (this app's default) so we can set it up for publishing lower priority messages which upsert
+ * Donations in Salesforce.
+ *
+ * It remains abstract so that it can be configured for either SQS or Redis in dependencies.php.
+ */
+abstract class SalesforcePushTransport implements TransportInterface
```

Comment:
```suggestion
 */
interface SalesforcePushTransport extends TransportInterface
```

But as discussed in other comments we never actually use this as a type declaration and I don't think anything concrete class was ever going to extend or implement this at runtime so probably better to use a plain string identifier instead of a classlike.